packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -105,15 +104,64 @@ + (NSString * _Nullable)version {
   return nil;
 }
 
+// Expo developers: Enable the below code by running
+//     export EX_DEV_LAUNCHER_URL=http://localhost:8090
+// before doing pod install. This will cause the controller to see if
+// the expo-launcher packager is running, and if so, use that instead of
+// the prebuild bundle.
+// See the pod_target_xcconfig definition in expo-dev-launcher.podspec
+
+- (NSURL * _Nullable)devLauncherURL
```

Comment:
We usually use `nullable` and `nonnull` 😉
```suggestion
- (nullable NSURL *)devLauncherURL
```

Reply:
Done